packages/docs/src/pages/tutorial/store/basic/index.mdx

```diff
@@ -3,20 +3,21 @@ title: Storing State
 layout: tutorial
 ---
 
-Applications need to store state to be useful. (Otherwise, they are just static pages.) 
```

Comment:
I kinda think this sentence is important for 2 types of devs - 
1. Those that come from static site generators
2. Beginners in this field

Plus I think a "conversation like" tone is required to keep it light and fun (but that's just me)

So I would leave this one in tact

@craigshoemaker  @mhevery what do you guys think?

Reply:
You got it! I added it back below.